subsys/usb/usb_device.c

```diff
@@ -461,6 +461,45 @@ static bool usb_get_descriptor(u16_t type_index, u16_t lang_id,
 	return found;
 }
 
+static bool set_endpoint(const struct usb_ep_descriptor *ep_desc)
```

Comment:
The problem here maybe that controller+driver may operate in high speed mode (usb_dc_dw.c) and this information (FS or HS) is lacking.

Reply:
I will add HS support after my high speed PR is merged #15140